check_url.py

```diff
@@ -37,35 +37,35 @@ def check_http(url, http_expectation):
         print('{} ✅ => {}'.format(url, http_got))
         return False
 
-
```

Comment:
Why!?

Reply:
Following code is never used (dead code). Therefore I've commented it.